subprojects/version-control/src/test/groovy/org/gradle/vcs/git/internal/GitVersionControlSystemSpec.groovy

```diff
@@ -150,6 +150,23 @@ class GitVersionControlSystemSpec extends Specification {
         thrown GradleException
     }
 
+    def 'error if repo is not a git repo'() {
+        given:
+        def target = tmpDir.file('versionDir')
+        repoSpec.url = 'https://notarepo.io'
```

Comment:
We should use [`.invalid`](https://en.wikipedia.org/wiki/.invalid) for this or another one of the test domain names

Reply:
Done.